doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-settings-changing-permissions-proc.adoc

```diff
@@ -1,15 +1,23 @@
 [id='business-central-settings-changing-permissions-proc_{context}']
-= Changing permissions for groups and roles
+= Changing permissions for groups and roles in {CENTRAL}
 
 You cannot change permissions for an individual user. However, you can change permissions for groups and roles. The changed permissions apply to users that have the role or belong to the group that you changed.
 
+[NOTE]
+====
+Any changes you make to roles or groups affect all of the users associated with that role or group.
+====
+
 .Procedure
 . In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen.
 . Click *Roles* or *Groups* to open the corresponding list.
 . From the list, click the role or group you want to edit. All details are displayed in the right pane.
 . Set the *Home Page* or *Priority* under the *Settings* section.
 . Set the {CENTRAL}, page, editor, space, and project permissions under the *Permissions* section.
 +
+.Setting the permissions
+image::Workbench/SecurityManagement/set-permissions.png[Setting the permissions]
++
 You can expand the resource type whose permissions you want to change.
 . Optional: To add an item not listed, click *Add Exception* and then set the permissions as required.
```

Comment:
@kaldesai I think "To add an item not listed" conveys a contrasting message, lets replace with something like "To add an exception for a resource type".

Reply:
Done